foreshadow/parallelprocessor.py

```diff
@@ -1,4 +1,7 @@
 """Foreshadow extension of feature union for handling dataframes."""
+# flake8: noqa
```

Comment:
We should remove this on do it on a per function basis. Not important for now but we should issue this so we don't forget.